dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/LibertyProviderImpl.java

```diff
@@ -53,13 +55,31 @@
     @Override
     public ServiceDelegate createServiceDelegate(URL url, QName qname,
                                                  @SuppressWarnings("rawtypes") Class cls) {
+        
+        //Eager initialize the StaxUtils
```

Comment:
Needed to update the client-side integration to allow CXF to use the STaX from the RI since that's what we ship with OL. We need to change this to use WoodStox in the future.